pom.xml

```diff
@@ -23,13 +25,14 @@
     <packaging>jar</packaging>
 
     <name>iban4j</name>
-    <description>A Java library for International Bank Account Number (IBAN) generation.</description>
-    <url>http://iban4j.org</url>
```

Comment:
This URL is broken for me, and doesn't load. It's also on the GitHub 'about' page. As a quick fix, I removed it here. 

<img width="434" alt="image" src="https://user-images.githubusercontent.com/897017/185556732-b89c72b6-4c86-4a0b-89e8-4695e317ec45.png">